scripts/genappimage.sh

```diff
@@ -69,14 +71,16 @@ cd "$APP_BUILD_DIR" # Get out of AppImage directory.
 #   - Expects: $ARCH, $APP, $VERSION env vars
 #   - Expects: ./$APP.AppDir/ directory
 #   - Produces: ../out/$APP-$VERSION.glibc$GLIBC_NEEDED-$ARCH.AppImage
-generate_appimage
+if [ -n "$TAG" ]; then
+  generate_type2_appimage -u "gh-releases-zsync|neovim|neovim|$TAG|nvim.appimage.zsync"
```

Comment:
so this is a passive operation, and by convention(?) `AppImageUpdate` somehow knows to look for the `*.zsync` file in our `releases/` ? Or does this contact AppImage servers?

Reply:
It's part of the [spec](https://github.com/AppImage/AppImageSpec/blob/35c50ecc519bdf499e174c1d1cd445b87a37f4e6/draft.md#update-information).  Adding the `-u ...` embeds extra information into the AppImage which AppImageUpdate knows to look for.  When AppImageUpdate sees this, it will check the github releases based on this information to see if an update is available.